src/components/Header/BaseHeader.jsx

```diff
@@ -57,6 +58,7 @@ const BaseHeader = ({
                 <a
                   className="rvt-lockup"
                   href={homeUrl}
+                  onClick={onClick}
```

Comment:
What if we handled the `preventDefault` behavior *only when* `onClick` is specified so end users don't have to do so?

Reply:
Done